outline/manager.js

```diff
@@ -490,8 +475,7 @@ module.exports = function (doc) {
                 ico.style = paneHiddenStyle
               }
 
-                // If the view already exists, remove it
-              state = 'paneShown'
+              // If the view already exists, remove it
```

Comment:
It's not quite clear to me what line this comment applies to, but if it was the line `state = 'paneShown'`, then I suppose this comment should be removed too :)